packages/luna/src/locales/ar-eg.json

```diff
@@ -277,6 +277,17 @@
   "dashboard.user.form.create": "مستخدم جديد",
   "dashboard.user.form.create.success": "تم إنشاء مستخدم جديد بنجاح",
   "dashboard.user.form.create.error": "نأسف، حدث خطأ أثناء إنشاء المستخم",
+  "dashboard.invoices.userId": "الرقم التعريفى للمستخدم",
+  "dashboard.invoices.method": "طريقة الدفع",
+  "dashboard.invoices.receiver": "المستقبل",
+  "dashboard.invoices.bank": "البنك",
+  "dashboard.invoices.amount": "الكمية",
+  "dashboard.invoices.status": "حالة الإيصال",
+  "dashboard.invoices.note": "ملاحظات",
+  "dashboard.invoices.attachment": "مرفقات",
+  "dashboard.invoices.addressedBy": "تم إصداره بواسطة",
+  "dashboard.invoices.createdAt": "تاريخ الإنشاء",
+  "dashboard.invoices.updatedAt": "تاريخ التحديث",
```

Comment:
I think `createdAt` and `updatedAt`  will be repeated a lot. Let's move it into a common place and reuse their text (please check if this trick is not already done).

Reply:
Done and updated it in everywhere it's needed
from now on they are in:
`global.createdAt`
`global.updatedAt`